### ISSUE TITLE
fake openssl-9999 package may need same USE flags as the actual one.

### ISSUE THREAD

**Issue by aleiphoenix** [gentoo/libressl#115]
I'm trying to use libressl to replace openssl on my system.

```
dev-libs/openssl:0

  (dev-libs/openssl-9999:0/0::libressl, installed) pulled in by
    (no parents that aren't satisfied by other packages in this slot)

  (dev-libs/openssl-1.0.2d:0/0::gentoo, ebuild scheduled for merge) pulled in by
    dev-libs/openssl:0[zlib?] required by (net-proxy/haproxy-1.5.14:0/0::gentoo, ebuild scheduled for merge)
```

This prevent haproxy to build.

see #116 .

**Comment by blueness** [comment]
I merged #116.  Is this still an issue?

**Comment by aleiphoenix** [comment]
Not anymore :) Thanks.
